apps/prairielearn/src/tests/groupRole.test.js

```diff
@@ -92,16 +92,18 @@ const leaveGroup = async (assessmentUrl) => {
  * @param {Array} roleAssignments
  * @param {String} assessmentId
  */
-const verifyRoleAssignmentsInDatabase = async (roleAssignments, assessmentId) => {
-  const expected = roleAssignments.map(({ roleId, groupUserId }) => ({
-    user_id: groupUserId,
-    group_role_id: roleId,
-  }));
+async function verifyRoleAssignmentsInDatabase(roleAssignments, assessmentId) {
+  const expected = roleAssignments
+    .map(({ roleId, groupUserId }) => ({
+      user_id: groupUserId,
+      group_role_id: roleId,
+    }))
+    .sort((a, b) => a.user_id - b.user_id || a.group_role_id - b.group_role_id);
```

Comment:
Sorting added since the change in the update process caused these entries to no longer have predictable return order, and since the order of these assignments doesn't matter.